en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md

```diff
@@ -0,0 +1,363 @@
+---
+layout: news_post
+title: "Ruby 2.7.0-rc1 Released"
+author: "naruse"
+translator:
+date: 2019-12-17 12:00:00 +0000
+lang: en
+breaks: false
+---
+
+{% assign release = site.data.releases | where: "version", "2.7.0-rc1" | first %}
+
+We are pleased to announce the release of Ruby 2.7.0-rc1.
+
+A preview version is released to gather feedback for the final release
```

Comment:
preview version?

Reply:
it should be fixed as "A release candidate is ..."

Reply:
fixed